<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(97, 97, 97);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>0,00 руб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#a71d5d; background-color:#f5f5f5;&quot;&gt;QPushButton&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#795da3; background-color:#f5f5f5;&quot;&gt;:pressed&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt; {&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;/&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#0086b3; background-color:#f5f5f5;&quot;&gt;border-style&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;: inset;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;/&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#0086b3; background-color:#f5f5f5;&quot;&gt;background&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;: qradialgradient(&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        radius: 1.35, stop: 0 #fff, stop: 1 #ddd&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        );&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;    }&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#a71d5d; background-color:#f5f5f5;&quot;&gt;QPushButton&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#795da3; background-color:#f5f5f5;&quot;&gt;:pressed&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt; {&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;/&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#0086b3; background-color:#f5f5f5;&quot;&gt;border-style&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;: inset;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;/&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#0086b3; background-color:#f5f5f5;&quot;&gt;background&lt;/span&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;: qradialgradient(&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        radius: 1.35, stop: 0 #fff, stop: 1 #ddd&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;        );&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Consolas','Courier New','monospace'; font-size:13px; color:#333333; background-color:#f5f5f5;&quot;&gt;    }&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border-radius: 20px;
border-style: outset;
padding: 5px;
QPushButton:hover</string>
    </property>
    <property name="text">
     <string>История расходов </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border-radius: 20px;
border-style: outset;
padding: 5px;
QPushButton:hover</string>
    </property>
    <property name="text">
     <string>История доходов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border-radius: 20px;
border-style: outset;
padding: 5px;
QPushButton:hover
</string>
    </property>
    <property name="text">
     <string>Добавить в бюджет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border-radius: 20px;
border-style: outset;
padding: 5px;
QPushButton:hover</string>
    </property>
    <property name="text">
     <string>Вычесть с бюджета</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border-radius: 20px;
border-style: outset;
padding: 5px;
QPushButton:hover</string>
    </property>
    <property name="text">
     <string>Конвентор</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
